sql: TestBackfillWithProtectedTS failed

sql.TestBackfillWithProtectedTS [failed](https://mesolite.cluster.engflow.com/invocations/default/21224f7b-bd6c-4dc4-830c-63571a1bc6a6?testReportRun=1&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvc3FsOnNxbF90ZXN0) on release-25.1 @ [db54d89df7aaf8f0ae77d476ca6daee76ad27006](https://github.com/cockroachdb/cockroach/commits/db54d89df7aaf8f0ae77d476ca6daee76ad27006):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestBackfillWithProtectedTS (15m0s)
	TestBackfillWithProtectedTS/create_index (14m11s)

```
Stack: 

```
goroutine 784475 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestBackfillWithProtectedTS
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestBackfillWithProtectedTS716008832
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
=== RUN   TestBackfillWithProtectedTS/create_index
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackfillWithProtectedTS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-46581

sql.TestBackfillWithProtectedTS [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18533221?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18533221?buildTab=artifacts#/) on release-25.1 @ [a27440708e2076b1084f36cd46f0d30fbc47b24d](https://github.com/cockroachdb/cockroach/commits/a27440708e2076b1084f36cd46f0d30fbc47b24d):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestBackfillWithProtectedTS (15m0s)
	TestBackfillWithProtectedTS/create_index (14m12s)

```
Stack: 

```
goroutine 553083 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestBackfillWithProtectedTS
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/ebab9da743cf8d3e9fbccfacc75c4814/logTestBackfillWithProtectedTS82436892
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
=== RUN   TestBackfillWithProtectedTS/create_index
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackfillWithProtectedTS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql.TestBackfillWithProtectedTS [failed](https://tanzanite.cluster.engflow.com/invocations/default/0f087c08-b16e-4281-a562-6c96ffb70d22?testReportRun=5&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvc3FsOnNxbF90ZXN0) on release-25.1 @ [f9c2a2f417a567a1dc6e7b23b41b8759cf62d077](https://github.com/cockroachdb/cockroach/commits/f9c2a2f417a567a1dc6e7b23b41b8759cf62d077):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestBackfillWithProtectedTS (15m0s)
	TestBackfillWithProtectedTS/create_index (14m13s)

```
Stack: 

```
goroutine 662032 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestBackfillWithProtectedTS
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestBackfillWithProtectedTS2578073983
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
=== RUN   TestBackfillWithProtectedTS/create_index
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=5</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackfillWithProtectedTS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql.TestBackfillWithProtectedTS [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18541577?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18541577?buildTab=artifacts#/) on release-25.1 @ [29c0a3d5161b6818e414efc9c35321c1c0416e19](https://github.com/cockroachdb/cockroach/commits/29c0a3d5161b6818e414efc9c35321c1c0416e19):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestBackfillWithProtectedTS (15m0s)
	TestBackfillWithProtectedTS/create_index (14m11s)

```
Stack: 

```
goroutine 540720 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestBackfillWithProtectedTS
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/ebab9da743cf8d3e9fbccfacc75c4814/logTestBackfillWithProtectedTS187755608
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
I250117 23:44:09.035188 1 (gostd) rand.go:246  [-] 1  random seed: 4039395864794345507
=== RUN   TestBackfillWithProtectedTS/create_index
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackfillWithProtectedTS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql.TestBackfillWithProtectedTS [failed](https://tanzanite.cluster.engflow.com/invocations/default/6d906e79-9f1c-4338-92cc-c8266d86a8c3?testReportRun=19&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvc3FsOnNxbF90ZXN0) on release-25.1 @ [29c0a3d5161b6818e414efc9c35321c1c0416e19](https://github.com/cockroachdb/cockroach/commits/29c0a3d5161b6818e414efc9c35321c1c0416e19):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestBackfillWithProtectedTS (15m0s)
	TestBackfillWithProtectedTS/create_index (14m26s)

```
Stack: 

```
goroutine 832677 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestBackfillWithProtectedTS
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestBackfillWithProtectedTS1987365850
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
=== RUN   TestBackfillWithProtectedTS/create_index
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=19</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackfillWithProtectedTS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql.TestBackfillWithProtectedTS [failed](https://tanzanite.cluster.engflow.com/invocations/default/eda20950-ce09-43f7-87c1-0b77653cbab9?testReportRun=6&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvc3FsOnNxbF90ZXN0) on release-25.1 @ [6123ebaa10c3847899aa81b09ec4cb8d08c0bd6d](https://github.com/cockroachdb/cockroach/commits/6123ebaa10c3847899aa81b09ec4cb8d08c0bd6d):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestBackfillWithProtectedTS (15m0s)
	TestBackfillWithProtectedTS/create_index (14m30s)

```
Stack: 

```
goroutine 734040 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestBackfillWithProtectedTS
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestBackfillWithProtectedTS2946269352
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
=== RUN   TestBackfillWithProtectedTS/create_index
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=6</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackfillWithProtectedTS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql.TestBackfillWithProtectedTS [failed](https://tanzanite.cluster.engflow.com/invocations/default/6a723732-b1cb-4f15-85f0-4a6c68bc048d?testReportRun=1&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvc3FsOnNxbF90ZXN0) on release-25.1 @ [6123ebaa10c3847899aa81b09ec4cb8d08c0bd6d](https://github.com/cockroachdb/cockroach/commits/6123ebaa10c3847899aa81b09ec4cb8d08c0bd6d):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestBackfillWithProtectedTS (15m0s)
	TestBackfillWithProtectedTS/create_index (14m19s)

```
Stack: 

```
goroutine 771068 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x385
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestBackfillWithProtectedTS
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestBackfillWithProtectedTS3246834527
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
=== RUN   TestBackfillWithProtectedTS/create_index
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #139493 sql: TestBackfillWithProtectedTS failed [C-test-failure O-robot T-sql-foundations branch-master release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackfillWithProtectedTS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql.TestBackfillWithProtectedTS [failed](https://tanzanite.cluster.engflow.com/invocations/default/5f767eae-57b7-4be9-9575-418e8ebd2dfe?testReportRun=3&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvc3FsOnNxbF90ZXN0) on release-25.1 @ [0a14af43960ce2fcb79a2624d3b94fe2ce3662e2](https://github.com/cockroachdb/cockroach/commits/0a14af43960ce2fcb79a2624d3b94fe2ce3662e2):


```
=== RUN   TestBackfillWithProtectedTS
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestBackfillWithProtectedTS691518875
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    backfill_protected_timestamp_test.go:518: -- test log scope end --
test logs left over in: outputs.zip/logTestBackfillWithProtectedTS691518875
--- FAIL: TestBackfillWithProtectedTS (488.87s)
=== RUN   TestBackfillWithProtectedTS/create_index
    backfill_protected_timestamp_test.go:472: running backfill with PTS not setup early enough
    backfill_protected_timestamp_test.go:425: i=0 begin running operations in the middle of backfill
    backfill_protected_timestamp_test.go:429: i=0 finished running operations in the middle of backfill
    backfill_protected_timestamp_test.go:507: 
        	Error Trace:	pkg/sql/backfill_protected_timestamp_test.go:507
        	Error:      	Received unexpected error:
        	            	pq: batch timestamp 1737566785.307652206,0 must be after replica GC threshold 1737566786.572575603,0 (r77: /Table/107{-/2})
        	Test:       	TestBackfillWithProtectedTS/create_index
    --- FAIL: TestBackfillWithProtectedTS/create_index (421.05s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=3</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #139493 sql: TestBackfillWithProtectedTS failed [C-test-failure O-robot P-1 T-sql-foundations branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackfillWithProtectedTS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql.TestBackfillWithProtectedTS [failed](https://tanzanite.cluster.engflow.com/invocations/default/29d1f443-1dd0-47e5-810d-9b0a3c6e1535?testReportRun=2&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvc3FsOnNxbF90ZXN0) on release-25.1 @ [0fb79d22b0fc0719f28a1e3993be2c4babf5155e](https://github.com/cockroachdb/cockroach/commits/0fb79d22b0fc0719f28a1e3993be2c4babf5155e):


```
=== RUN   TestBackfillWithProtectedTS
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestBackfillWithProtectedTS73121702
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    backfill_protected_timestamp_test.go:518: -- test log scope end --
test logs left over in: outputs.zip/logTestBackfillWithProtectedTS73121702
--- FAIL: TestBackfillWithProtectedTS (503.82s)
=== RUN   TestBackfillWithProtectedTS/create_index
    backfill_protected_timestamp_test.go:472: running backfill with PTS not setup early enough
    backfill_protected_timestamp_test.go:425: i=0 begin running operations in the middle of backfill
    backfill_protected_timestamp_test.go:429: i=0 finished running operations in the middle of backfill
    backfill_protected_timestamp_test.go:507: 
        	Error Trace:	pkg/sql/backfill_protected_timestamp_test.go:507
        	Error:      	Received unexpected error:
        	            	pq: batch timestamp 1737651857.566750550,0 must be after replica GC threshold 1737651890.244480675,0 (r80: /{Table/107/3-Max})
        	Test:       	TestBackfillWithProtectedTS/create_index
    --- FAIL: TestBackfillWithProtectedTS/create_index (466.33s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=2</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #139493 sql: TestBackfillWithProtectedTS failed [C-test-failure O-robot P-1 T-sql-foundations branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackfillWithProtectedTS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql.TestBackfillWithProtectedTS [failed](https://tanzanite.cluster.engflow.com/invocations/default/34d26803-28e5-4b40-a454-7c89b2b21441?testReportRun=12&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvc3FsOnNxbF90ZXN0) on release-25.1 @ [98ef65420f236cb16e7d88767ce3e38eab1e9b54](https://github.com/cockroachdb/cockroach/commits/98ef65420f236cb16e7d88767ce3e38eab1e9b54):


```
=== RUN   TestBackfillWithProtectedTS
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestBackfillWithProtectedTS4025971303
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:139: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    backfill_protected_timestamp_test.go:526: -- test log scope end --
test logs left over in: outputs.zip/logTestBackfillWithProtectedTS4025971303
--- FAIL: TestBackfillWithProtectedTS (494.28s)
=== RUN   TestBackfillWithProtectedTS/create_index
    backfill_protected_timestamp_test.go:477: running backfill with PTS not setup early enough
    backfill_protected_timestamp_test.go:425: i=0 begin running operations in the middle of backfill
    backfill_protected_timestamp_test.go:429: i=0 finished running operations in the middle of backfill. retErr: failed to DELETE and INSERT: pq: batch timestamp 1737748197.216276518,0 must be after replica GC threshold 1737748233.515274144,0 (r77: /Table/107{-/2})
    backfill_protected_timestamp_test.go:474: backfill goroutine failed: could not read jobs table: context canceled
    backfill_protected_timestamp_test.go:515: 
        	Error Trace:	pkg/sql/backfill_protected_timestamp_test.go:515
        	Error:      	Received unexpected error:
        	            	failed to DELETE and INSERT: pq: batch timestamp 1737748197.216276518,0 must be after replica GC threshold 1737748233.515274144,0 (r77: /Table/107{-/2})
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/sql_test.TestBackfillWithProtectedTS.func7.2.1
        	            	  | 	pkg/sql/backfill_protected_timestamp_test.go:438
        	            	  | github.com/cockroachdb/cockroach/pkg/sql_test.TestBackfillWithProtectedTS.func7.2
        	            	  | 	pkg/sql/backfill_protected_timestamp_test.go:463
        	            	  | github.com/cockroachdb/cockroach/pkg/sql_test.TestBackfillWithProtectedTS.func7.Group.GoCtx.4
        	            	  | 	pkg/util/ctxgroup/ctxgroup.go:189
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	external/org_golang_x_sync/errgroup/errgroup.go:78
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1695
        	            	Wraps: (2) failed to DELETE and INSERT
        	            	Wraps: (3) pq: batch timestamp 1737748197.216276518,0 must be after replica GC threshold 1737748233.515274144,0 (r77: /Table/107{-/2})
        	            	Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *pq.Error
        	Test:       	TestBackfillWithProtectedTS/create_index
    --- FAIL: TestBackfillWithProtectedTS/create_index (461.61s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=12</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #139775 sql: TestBackfillWithProtectedTS failed [C-test-failure O-robot T-sql-foundations branch-release-24.1 release-blocker]
- #139493 sql: TestBackfillWithProtectedTS failed [C-test-failure O-robot P-1 T-sql-foundations branch-master]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackfillWithProtectedTS.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/backfill_protected_timestamp_test.go
+++ b/pkg/sql/backfill_protected_timestamp_test.go
@@ -397,10 +397,10 @@ func TestBackfillWithProtectedTS(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			for _, sql := range []string{
 				"SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false",
-				"ALTER DATABASE defaultdb CONFIGURE ZONE USING gc.ttlseconds = 1",
+				"ALTER DATABASE defaultdb CONFIGURE ZONE USING gc.ttlseconds = 5",
 				"DROP TABLE IF EXISTS t CASCADE",
 				"CREATE TABLE t(n int)",
-				"ALTER TABLE t CONFIGURE ZONE USING range_min_bytes = 0, range_max_bytes = 67108864, gc.ttlseconds = 1",
+				"ALTER TABLE t CONFIGURE ZONE USING range_min_bytes = 0, range_max_bytes = 67108864, gc.ttlseconds = 5",
 				fmt.Sprintf("INSERT INTO t(n) SELECT * FROM generate_series(1, %d)", initialRowCount),
 			} {
 				r.Exec(t, sql)

